docs/en/Extending-Existing-Entities-Core.md

```diff
@@ -0,0 +1,232 @@
+
+```csharp
+public class UserListDto : EntityDto<long>, IPassivable, IHasCreationTime
+{
+    //...existing code
+
+    public string Address { get; set; }
+}
+```
+
+Since UserListDto **auto maps** from User entity, no need to change **UserAppService.GetUsers** method. Now, we can go to UI side to add Address property to the **users table**. Open 
+.Web\\wwwroot\\view-resources\Areas\\{AppAreaName}\Views\\Users\\**Index.js** and add Address column right after CreationTime column:
+
+```javascript
+                },
+                {
+                    targets: 9,
+                    data: "creationTime",
+                    render: function (creationTime) {
+                        return moment(creationTime).format('L');
+                    }
+                },
+                {
+                    targets: 10,
+                    data: 'address'
+                }
+            ]
+        });
+```
+
+And in .Web\\Areas\\{AppAreaName}\Views\\Users\\**Index.cshtml**, add Address column:
+
+
+```html
+                            <th>@L("Active")</th>
+                            <th>@L("CreationTime")</th>
+                            <th>@L("Address")</th>
+                        </tr>
+                    </thead>
+                </table>
+```
+
+That's all. Let's run the application and open the **users page**:
+
+<img src="images/extend-entities-user-address-in-table-2.png" alt="Address in table" class="img-thumbnail" width="1381" height="478" />
+
+### Add Address On User Create/Edit
+
+We may want to set Address while **creating/editing** a User.
+
+Clients use UserAppService.**GetUserForEdit** method to show user information on edit form. It returns **GetUserForEditOutput** object which contains a **UserEditDto** object that includes user properties.
+So, we should add Address to UserEditDto to allow clients to change Address property on create/update:
+
+```csharp
+public class UserEditDto : IPassivable
+{
+    //...existing code
+
+    public string Address { get; set; }
+}
+```
+
+Since **UserAppService** use **auto mapping**, no need to manually map Address to the User entity. So, server side code is just that. We can go to UI side to add an Address field to the form:
+
+```html
+                        <div class="form-group">
+                            <label for="Address">@L("Address")</label>
+                            <input id="Address" type="text" name="Address" class="form-control" value="@Model.User.Address">
+                        </div>
+```
+
+This code is written to .Web\\Areas\\{AppAreaName}\Views\\Users\\createOrEditModal.cshtml**. After adding, new Address field is shown on the create/edit form as shown below:
```

Comment:
There is usage of "**" after createOrEditModal.cshtml but there is no beginning of this mark so nothing is showed in bold when viewing the document.